docs/manual/upgrades.md

```diff
@@ -0,0 +1,129 @@
+In this case the operator is not able to handle the upgrade correctly since the returned cluster status has no information besides the coordinators that are reachable.
+During this time the FDB cluster will be unavailable.
+You have to manually restart all processes with the [kubectl fdb plugin](../../kubectl-fdb/Readme.md) with the command from above.
+
+#### Processes are not restarted during the restart Phase
+
+This issue is similar to the one above with the difference that affected processes are not restarted and they are still running on the old `fdbserver` version.
+For version incompatible upgrades that means that those processes are not reporting to the cluster.
+The operator will mark this Process Group with the `MissingProcesses` condition, since the process will be missing in the [cluster status json](https://apple.github.io/foundationdb/mr-status.html).
+If those processes are reported in the `incompatible_connections` field of the cluster status json the operator will recreate those Pods directly with the new image in the `RemoveIncompatibleProcesses` subreconciler.
+Otherwise the Pod will recreate those Pods in the [recreation of Pods phase](#recreation-of-pods-phase) with the new image.
+During that time the cluster might run with reduced capacity since not all processes are reporting to the cluster.
+
+The operator can handle those cases automatically.
+If you want to speed up this process you can restart the processes with the [kubectl fdb plugin](../../kubectl-fdb/Readme.md):
+
+```bash
+kubectl fdb restart -c cluster --process-condition=MissingProcesses
+```
+
+One special case here is when the majority of coordinators are not restarted and they are stuck on the old version.
+In that case the operator is not able to handle this case automatically and you have to restart the coordinator processes manually with the [kubectl fdb plugin](../../kubectl-fdb/Readme.md).
+Either by targeting directly the coordinator Pods (you can get the coordinator Pods by comparing the IPs of the connection string with the IP addresses of the Pods):
+
+```bash
+kubectl fdb -n default restart -c cluster pod-1 pod-2
+```
+
+or by restarting all processes of the cluster:
+
+```bash
+kubectl fdb restart -c cluster --all-processes
+```
+
+#### Detect correct version
+
+The operator tries per default to read the [cluster status json](https://apple.github.io/foundationdb/mr-status.html) with the [multi-version](https://apple.github.io/foundationdb/api-general.html#multi-version-client) from the system key-space.
+During an upgrade it can happen that the multi-version client tries to connect to the cluster with the wrong version especially if not all coordinators are upgraded.
+In this case the operator will fallback using `fdbcli` for the different versions (the one that is desired and the current running version).
```

Comment:
My guess is that this works only when a majority of coordinators are on the "desired" version).

Reply:
Good point. I add that 👍